packages/hasher.go

```diff
@@ -0,0 +1,64 @@
+package packages
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sonatype-nexus-community/go-sona-types/cyclonedx"
+)
+
+type Hasher struct {
+	logLady *logrus.Logger
+	Files   []cyclonedx.File
+}
+
+func New(logLady *logrus.Logger) *Hasher {
+	return &Hasher{logLady: logLady}
+}
+
+func (h Hasher) PopulateListOfHashes(path string) {
+	for _, v := range filepath.SplitList(path) {
+		err := filepath.Walk(v, func(path string, f os.FileInfo, err error) error {
```

Comment:
minor: maybe use a named error for the func() pointer, e.g. `func(path string, f os.FileInfo, err error) (hashErr error) {`...